docs/architecture/transaction.md

```diff
@@ -0,0 +1,64 @@
+# Transaction
+
+> Single account state transition
+
+## What is the purpose of a transaction?
+
+To facilitate single account state transitions.
+
+## What is a transaction?
+
+In Miden, a `Transaction` represents the state transition of a single account. A `Transaction` takes a single [account](accounts.md) and one or more [notes](notes.md), and none or one script (piece of code executed after all notes have been executed) as input, and outputs the same account with a potentially updated state, together with some potential newly created notes.
+
+![Transaction diagram](../img/architecture/transaction/transaction-diagram.png)
```

Comment:
To me this digram is somewhat confusing. I would use something simpler (which have a simpler version in the deck). The proportions of the graphic are also off (it is too big).